training/Containerfile

```diff
@@ -0,0 +1,14 @@
+FROM nvcr.io/nvidia/cuda:12.4.1-devel-ubi9
+RUN dnf install -y python3.11 python3.11-devel git python3-pip make automake gcc gcc-c++
```

Comment:
@sallyom do we still need to a clean all here?

Reply:
yea, i believe we should add `&& dnf clean all` to that RUN cmd